tests/foreman/ui/test_contenthost.py

```diff
@@ -370,6 +370,49 @@ def test_positive_provisioning_host_link(self):
             self.assertEqual(self.browser.current_url, host_url)
 
     @tier3
+    def test_positive_ensure_errata_applicability_with_host_reregistered(self):
```

Comment:
can we have this test for upgrade aswell , coud you please add  upgrade decorator to test